linera-storage/src/lib.rs

```diff
@@ -278,41 +280,57 @@ pub trait Storage: Sized {
         let compressed_contract_bytecode = CompressedBytecode {
             compressed_bytes: contract_blob.into_bytes().to_vec(),
         };
-        let contract_bytecode =
+        let _contract_bytecode =
```

Comment:
Could we add an attribute instead of the underscore. The pattern is to add the underscore to unused bindings, but in this case it might be used.

```suggestion
        #[cfg_attr(not(any(with_wasm_runtime, with_revm)), allow(unused_variables))]
        let contract_bytecode =
```

Reply:
Thank you.